app/main.js

```diff
@@ -930,9 +979,12 @@ function createPreferencesWindow () {
   preferencesWin.on('closed', () => {
     preferencesWin = null
   })
+  setTimeout(() => {
+    preferencesWin.center()
+  }, 0)
 }
 
-function updateTray () {
+async function updateTray () {
```

Comment:
Just to know for myself: why was this needed?

Reply:
In previous version I used image generation and generations could take time. I do not wanted to return path that was not ready coz it would not work. Anyway in current implementation it is not needed and could only be needed when short break would be implemented not by brute force approach.